kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -124,4 +127,19 @@ export default [
       }
     },
   },
+  {
+    path: '/:facility_id?/managesync',
+    component: ManageSyncSchedule,
+    name: 'ManageSyncSchedule',
+  },
+  {
+    path: '/:facility_id?/editsyncschedule/?',
+    component: editDeviceSyncSchedule,
+    name: 'EditSyncSchedule',
+  },
+  {
+    path: '/:facility_id?/editdevice',
```

Comment:
I think it would be `'/:facility_id?/:device_id/edit'` without the `?` because `deviceId` is a required parameter. At the moment you are accessing this via a query parameter instead, `$route.query.deviceId` whereas you would access this by `$route.params.deviceId`.